### PR TITLE
PWN-7587 - THS update token each time login

### PR DESCRIPTION
**Pull request by NikolaiEmelianov**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-7587

## Description of Work

update token each time login

### CODE REVIEW

File: app/src/main/java/org/p2p/wallet/push_notifications/AppFirebaseMessagingService.kt [hunk 1]
```diff
@@ -19,10 +22,13 @@ class AppFirebaseMessagingService : FirebaseMessagingService(), KoinComponent {
     private val appNotificationManager: AppNotificationManager by inject()
     private val appsFlyerService: AppsFlyerService by inject()
     private val intercomPushService: IntercomPushService by inject()
+    private val pushNotificationsInteractor: PushNotificationsInteractor by inject()
+    private val appScope: AppScope by inject()
 
     override fun onNewToken(token: String) {
         appsFlyerService.onNewToken(token)
         intercomPushService.registerForPush(token)
+        appScope.launch { pushNotificationsInteractor.updateDeviceToken() }
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/push_notifications/AppFirebaseMessagingService.kt:31]: does onNewToken is called on every app launch?

Comment (NikolaiEmelianov) [reply app/src/main/java/org/p2p/wallet/push_notifications/AppFirebaseMessagingService.kt:31]: no

Comment (NikolaiEmelianov) [reply app/src/main/java/org/p2p/wallet/push_notifications/AppFirebaseMessagingService.kt:31]: signInByPinCode, signInByBiometric does

Comment (gslevinkov) [reply app/src/main/java/org/p2p/wallet/push_notifications/AppFirebaseMessagingService.kt:31]: oh, thanks!

**Review by gslevinkov — APPROVED** [review]

**Pull request merged**